audio/beets/Portfile

```diff
@@ -0,0 +1,601 @@
+
+checksums           rmd160  b78e41d04dc9a9199696e5c31d885e3503b54d2a \
+                    sha256  887f7bbac0fc14c49469e50d406fd216f914a27acf3818c6503c223f9825342b \
+                    size    1694845
+
+python.default_version 39
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+if {${name} eq ${subport}} {
+    depends_lib-append \
+                    port:py${python.version}-Pillow \
+                    port:py${python.version}-acoustid \
+                    port:py${python.version}-beautifulsoup4 \
+                    port:py${python.version}-confuse \
+                    port:py${python.version}-discogs-client \
+                    port:py${python.version}-flask \
+                    port:py${python.version}-flask-cors \
+                    port:py${python.version}-gmusicapi \
+                    port:py${python.version}-jellyfish \
+                    port:py${python.version}-langdetect \
+                    port:py${python.version}-last \
+                    port:py${python.version}-mediafile \
+                    port:py${python.version}-mpd2 \
+                    port:py${python.version}-munkres \
+                    port:py${python.version}-musicbrainzngs \
+                    port:py${python.version}-mutagen \
+                    port:py${python.version}-rarfile \
+                    port:py${python.version}-reflink \
+                    port:py${python.version}-requests \
+                    port:py${python.version}-requests-oauthlib \
+                    port:py${python.version}-six \
+                    port:py${python.version}-soco \
+                    port:py${python.version}-unidecode \
+                    port:py${python.version}-xdg \
+                    port:py${python.version}-yaml
+
+    depends_run-append \
+                    port:chromaprint \
+                    port:ffmpeg \
+                    port:flac \
+                    port:mp3val \
+                    port:ImageMagick \
+                    port:ipfs
+
+    post-destroot {
+        xinstall -d ${destroot}${prefix}/share/man/man1
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.1] ${destroot}${prefix}/share/man/man1/
+
+        xinstall -d ${destroot}${prefix}/share/man/man5
+        xinstall -m 0644 {*}[glob ${worksrcpath}/man/*.5] ${destroot}${prefix}/share/man/man5/
+
+        xinstall -d ${destroot}${prefix}/share/${name}
+        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE ${destroot}${prefix}/share/${name}
+    }
+} else {
+    depends_lib-append \
+                    port:beets
+
+    post-destroot {
+        foreach f {__init__.py __init__.pyc __init__.pyo __pycache__} {
+            set fp "${destroot}${python.pkgd}/beetsplug/$f"
+            if {[file exists ${fp}]} {
+                file delete -force ${fp}
+            }
+        }
+    }
+}
+
+subport beets-alternatives {
+    name            beets-alternatives
```

Comment:
for every subport you get a `port lint` message like: `Error: Portfile directory beets does not match port name beets-alternatives`. 

You should remove the `name` field in every subport as that is derived from the `subport <name>` already and if you do, it is fine with being in the Portfile in the `beets` directory.